### ISSUE TITLE
Secrets encrypted using KMS manged keys and not KMS CMK

### ISSUE THREAD

**Issue by zsaltys** [awslabs/aws-dataall#880]
There are a few secrets encrypted using KMS managed keys and not KMS CMK keys. The main disadvantage of KMS managed is that we cannot track in audit trail who is accessing encryption keys. This is being picked up by Checkov security scanner.

We would like to make sure that these and any other secrets are encrypted with KMS CMK keys.

```
CheckID		: CKV_AWS_149
CheckName	: Ensure that Secrets Manager secret is encrypted using KMS CMK
File		: /cdk.out/dataallcdktemplatecicdstackAurora45A3670B.nested.template.json:42-74
Resource	: AWS::SecretsManager::Secret.dataallcdktemplateauroradb624DC06D
Guideline	:[ CKV_AWS_149 ](https://docs.paloaltonetworks.com/content/techdocs/en_US/prisma/prisma-cloud/prisma-cloud-code-security-policy-reference/aws-policies/aws-general-policies/ensure-that-secrets-manager-secret-is-encrypted-using-kms.html)
```

```
CheckID		: CKV_AWS_149
CheckName	: Ensure that Secrets Manager secret is encrypted using KMS CMK
File		: /cdk.out/assembly-dataall-cdk-template-cicd-stack-dataall-staging-backend-stage/dataallcdktemplatecicdstackdataallstagingbackendstagebackendstackAuroraC9027C58.nested.template.json:90-122
Resource	: AWS::SecretsManager::Secret.dataallstagingauroradbDB096810
Guideline	: CKV_AWS_149
```

**Comment by noah-paige** [comment]
Both of these scanner outputs I believe to refer to DatabaseSecrets CDK construct in `dataall/deploy/stacks/aurora.py`

Specifying our own KMS CMK Key rather than using KMS managed is supported and I agree it would be good to ensure auditability [docs](https://docs.aws.amazon.com/cdk/api/v2/python/aws_cdk.aws_rds/DatabaseSecret.html).

Should be good to pick this up shortly barring any capacity setbacks, thank @zsaltys

**Comment by mourya-33** [comment]
@noah-paige I created the PR - https://github.com/awslabs/aws-dataall/pull/935 for the changes

**Comment by noah-paige** [comment]
Addressed PR #935 - PR is merged, thanks for opening this issue
